ambry-server/src/main/java/com/github/ambry/server/RepairRequestsSender.java

```diff
@@ -248,6 +281,16 @@ private List<RequestInfo> getAmbryRequest(long partitionId) throws Exception {
     return requestInfos;
   }
 
+  /**
+   * Get partitions which have RepairRequests to fix
+   * @return list of partition ids.
+   */
+  Set<Long> getPartitionIdsNeedRepair() throws SQLException {
+    // each server has 12 * 23 partition. right now search in one single query.
+    Set<Long> partitionsNeedRepair = db.getPartitionsNeedRepair(nodeId.getHostname(), nodeId.getPort(), partitionIds);
+    return partitionsNeedRepair;
```

Comment:
nit: just return the set, we don't need to assign it to another variable.

Reply:
Changed